src/en/mangago/src/eu/kanade/tachiyomi/extension/en/mangago/Mangago.kt

```diff
@@ -151,11 +151,17 @@ class Mangago : ParsedHttpSource() {
 
     override fun searchMangaNextPageSelector() = genreListingNextPageSelector
 
+    private val descriptionRegex = Regex("""The following content is intended for mature audiences and may contain sexual themes, gore, violence and/or strong language\. Discretion is advised\.""")
+    private fun cleanDescription(description: String) = description.replace(descriptionRegex, "").trim()
+
+    private val titleRegex = Regex("""\(yaoi\)|\{Official\}|«Official»|〘Official〙|\(Official\)|\s\[Official]|\s「Official」|『Official』|\s?/Official\b""")
+    private fun cleanTitle(title: String) = title.replace(titleRegex, "").substringBeforeLast("(").trim()
+
     override fun mangaDetailsParse(document: Document) = SManga.create().apply {
-        title = document.selectFirst(".w-title h1")!!.text()
+        title = cleanTitle(document.select(".w-title h1").text())
         document.getElementById("information")!!.let {
             thumbnail_url = it.selectFirst("img")!!.attr("abs:src")
-            description = it.selectFirst(".manga_summary")!!.text()
+            description = cleanDescription(it.selectFirst(".manga_summary")!!.text())
```

Comment:
the additional text is inside a `font` tag, you can remove it directly with jsoup. [example](https://github.com/keiyoushi/extensions-source/blob/main/lib-multisrc/fuzzydoodle/src/eu/kanade/tachiyomi/multisrc/fuzzydoodle/FuzzyDoodle.kt#L190-L192)